cdi-embedder/src/test/java/org/commonjava/maven/galley/embed/TestCDIProvider.java

```diff
@@ -270,9 +306,9 @@ public String getNodeId()
 
     @Produces
     @Default
-    public DefaultTrafficClassifier getTrafficClassifier()
+    public AbstractTrafficClassifier getTrafficClassifier()
```

Comment:
For the naming here, not sure if an abstract class as result of a producer method is good option. I'm always feeling if an interface is more proper. But this is not a big deal.

Reply:
Yes, i noticed that also and also default annotation is quastionable.

Reply:
Hm, I would agree. I'll look at why it's not using the interface there.

Reply:
I had to make a very minor change to o11yphant to resolve a remaining reference to the abstract classifier, in the GoldenSignalsFilter. I've done that here: https://github.com/Commonjava/o11yphant/pull/56